scripts/config-dhcp.sh

```diff
@@ -1,49 +1,90 @@
 #!/bin/bash
 
-if dpkg-query -W -f='${Status}' isc-dhcp-server 1>/dev/null; then
+#Check if isc-dhcp-server exists, if not install it.
+if dpkg-query -W -f='${Status} ${Version}' isc-dhcp-server | grep 4.3; then
 	echo "Already installed"
-	sudo apt-get upgrade isc-dhcp-server
+	sudo dpkg --configure -a 
 else
+	echo "Installing isc-dhcp-server"
 	sudo apt-get update --yes
-	sudo apt-get install isc-dhcp-server --yes
+	sudo apt-get install isc-dhcp-server=4.3.* --yes
+	sudo dpkg --configure -a
 fi
```

Comment:
check if is installed after the installation process, if the `apt install` fail, the script will continue, and this can not happen.